tests/Selector/AbstractNamedSelector.php

```diff
@@ -6,7 +6,7 @@
 use Behat\Mink\Selector\Xpath\Escaper;
 use PHPUnit\Framework\TestCase;
 
-abstract class NamedSelectorTest extends TestCase
+abstract class AbstractNamedSelector extends TestCase
```

Comment:
What was the purpose of this class renaming?

Reply:
PHPUnit 10 will raise a warning, if there are classes with the Suffix `Test` and no tests or abstract Classes. https://github.com/sebastianbergmann/phpunit/issues/4979#issuecomment-1144781859
https://github.com/sebastianbergmann/phpunit/issues/4979#issuecomment-1461594789

Reply:
The CI Tests failed because of this

Reply:
I would actually use `TestCase` as suffix here

Reply:
ok, I removed the prefix and added your suggested suffix.